src/main/webapp/app/exam/manage/exam-participant-scores/exam-participant-scores.component.html

```diff
@@ -0,0 +1,12 @@
+<jhi-alert></jhi-alert>
+
+<div *ngIf="isLoading" class="d-flex justify-content-center">
+    <div class="spinner-border" role="status">
+        <span class="sr-only">{{ 'loading' | translate }}</span>
+    </div>
+</div>
+
+<div *ngIf="!isLoading && participantScores">
+    <button class="btn btn-info mb-2" (click)="loadData()">Refresh</button>
+    <jhi-participant-scores [isLoading]="isLoading" [participantScores]="participantScores"></jhi-participant-scores>
+</div>
```

Comment:
This is same with course-participant-scores.component.html so a common child component would be nice for this.

Reply:
done